lablib/processors/ayon_hiero_effect_file.py

```diff
@@ -99,9 +99,9 @@ def load(self) -> None:
             class_obj = class_obj.from_node_data(node_value)
 
             # separate color ops from repo ops
-            if "color" in class_obj.__class__.__module__:
+            if isinstance(class_obj, operators.ColorOperator):
                 self._color_ops.append(class_obj)
-            else:
+            if isinstance(class_obj, operators.RepositionOperator):
```

Comment:
```suggestion
            elif isinstance(class_obj, operators.RepositionOperator):
```

Reply:
this one i'm not sure about. doesn't change the logic does it?

Reply:
Hum the previous code `if-else` ensure that `class_obj` was either part of `self._color_ops` or `self.repo_ops`.
The 2 consecutives `if` do not prevent that anymore of a class inherit from both, the `elif` suggestion restores that.

Reply:
previously it checked from which file the object was import from by string.
the whole point is to append to `self._color_ops` and `self._repo_ops`.
also currently there is no operator that inherits from `ColorOperator` *and* `RepositionOperator` and i somehow doubt there will be. 
so not sure which behaviour gets restored but really, fine to me ✌️